aten/src/ATen/native/Blas.cpp

```diff
@@ -33,7 +33,7 @@ TORCH_META_FUNC(addmv)(const Tensor &self, const Tensor &mat, const Tensor &vec,
     "vector + matrix @ vector expected, got ", self.dim(), ", ", mat.dim(), ", ", vec.dim());
 
   TORCH_CHECK(mat.size(1) == vec.size(0) && (mat.size(0) == self.numel() || self.numel() == 1),
-     "size mismatch, got ", self.size(0), ", ", mat.size(0), "x", mat.size(1), ",", vec.size(0));
+    "size mismatch, got bias (", self.size(0), "), matrix (", mat.size(0), "x", mat.size(1), "), vector (", vec.size(0), ")");
```

Comment:
I think it is ok to assume that this will be mostly for python side users. Could you update the names to be the ones from the python call (input, mat, vec): https://pytorch.org/docs/stable/generated/torch.addmv.html?highlight=addmv#torch.addmv

Reply:
That makes sense! Done